src/js/hca/config/form.js

```diff
@@ -369,6 +366,7 @@ const formConfig = {
           path: 'military-service/service-information',
           title: 'Service periods',
           uiSchema: {
+            'ui:description': __BUILDTYPE__ !== 'production' ? MilitaryPrefillMessage : undefined,
```

Comment:
Because we can't pre-fill military history outside of production?

Reply:
We actually can and do, but this message is just being shown in staging for user testing.